tempest/command_argument_string.py

```diff
@@ -45,6 +45,9 @@
 restore_security_groups = "workloadmgr restore-security-groups "
 network_restore = "workloadmgr restore-network-topology "
 
+#openstack secret commands
+openstack_create_secret_with_empty_payload = "openstack secret store -p \"\" -f json --insecure "
```

Comment:
Insecure flag should not be added to the command here. Instead it should be appended during execution based on AUTH_URL. This should be handled in fetch_resources.sh